cmd/session-clear.go

```diff
@@ -0,0 +1,157 @@
+/*
+ * Minio Client (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"encoding/json"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/minio/pkg/probe"
+)
+
+var sessionClearFlags = []cli.Flag{
+	cli.BoolFlag{
+		Name:  "force",
+		Usage: "Force a dangerous clear operation.",
+	},
+}
+
+var sessionClear = cli.Command{
+	Name:            "clear",
+	Usage:           "Clear a previously saved session.",
+	Action:          mainSessionClear,
+	Before:          setGlobalsFromContext,
+	Flags:           append(sessionClearFlags, globalFlags...),
+	HideHelpCommand: true,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+  
+USAGE:
+  {{.HelpName}} SESSION-ID|all
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Clear session.
+	 $ {{.HelpName}} clear ygVIpSJs
+  2. Clear all sessions.
+	 $ {{.HelpName}} clear all
+  3. Forcefully clear an obsolete session.
+	 $ {{.HelpName}} --force
```

Comment:
@vadmeste yes you are right this needs a session name. --force flag was added so that user is aware that files are obsolete. Before a clear would transparently delete these files without telling user they were no longer useful. 

What do you mean with providing a random session id?